app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from __future__ import annotations
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccine_error_msg = []
+    mask_error_msg = []
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as e:
+            print("You have some issue", e)
```

Comment:
The message printed here is not very informative. Consider changing it to something like `print(f"Vaccine error for {friend['name']}: {e}")` to provide more context.